aea/protocols/fipa/serialization.py

```diff
@@ -40,31 +40,31 @@ def encode(self, msg: Message) -> bytes:
 
         performative_id = msg.get("performative").value
         if performative_id == "cfp":
-            performative = fipa_pb2.FIPAMessage.CFP()
+            performative = fipa_pb2.FIPAMessage.CFP()  # type: ignore
```

Comment:
This is fine for the moment. However, I'd like to share this project: [MyPy Protobuf](https://github.com/dropbox/mypy-protobuf)

 `mypy-protobuf` is an open-source tool to generate MyPy stubs from Protobuf files. I haven't investigated in deep, but I believe that might help for all the Protobuf-based protocol serializations we have developed, and future ones.

Reply:
Good point!